src/mypage/components/MypageSubMenu.tsx

```diff
@@ -14,33 +20,45 @@ const MypageSubMenu = () => {
     openConfirm({
       text: WITHDRAWAL_MESSAGE,
       okText: "탈퇴하기",
+      isHeaderCloseBtn: true,
     });
   };
 
   const handleRoute = (url: string) => {
     push(url);
   };
 
+  const MENU_SUB: SubMenu[] = [
+    {
+      value: "문의하기",
+      onClick: () => handleContact(),
+    },
+    {
+      value: "공지사항",
+      onClick: () => handleRoute("/notice"),
```

Comment:
```suggestion
      onClick: () => push("/notice"),
```

[Lv2] push만 사용해도 될것같습니다!